.changeset/polite-bottles-relate.md

```diff
@@ -0,0 +1,5 @@
+---
+"openapi-zod-client": minor
+---
+
+Fix #260 by infer types of items in a required only allOf item.
```

Comment:
could you add an example (that you can take from one of the changed tests) and inline it in that changeset so that we can easily see what's different with this PR (from the future changelog) ?

Reply:
I added a test: lib/tests/allOf-infer-required-only-item.test.ts, would that help?